v2/AdminCore.WebApp/Controllers/Holiday/HolidayController.cs

```diff
@@ -0,0 +1,101 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="HolidayController.cs" company="AdminCore">
+//   AdminCore
+// </copyright>
+// <summary>
+//   Defines the HolidayController type.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+using System;
+using AdminCore.Common.Interfaces;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using AdminCore.WebApi.Models.Event;
+
+namespace AdminCore.WebApi.Controllers
+{
+  [Authorize]
+  [Route("[controller]")]
+  [ApiController]
+  public class HolidayController : ControllerBase
+  {
+    private readonly IMapper _mapper;
+
+    private readonly IEventService _eventService;
+
+    public HolidayController(IEventService holidayService, IMapper mapper)
+    {
+      _eventService = holidayService;
+      _mapper = mapper;
+    }
+
+    [HttpGet("/")]
+    public List<EventViewModel> GetAllHolidays()
+    {
+      throw new System.NotImplementedException();
+    }
+
+    [HttpGet("/{holidayId}")]
+    public EventViewModel GetHolidayByHolidayId(int holidayId)
+    {
+      throw new System.NotImplementedException();
+    }
+
+    [HttpGet("/findByEmployeeId/{employeeId}}")]
```

Comment:
Need to up do the routes to get instead of find

Reply:
Asked Mark this yesterday and he suggested keeping the routes as they are so we don't break front end